extensions/hibernate-validator/deployment/src/main/java/io/quarkus/hibernate/validator/deployment/MethodValidatedAnnotationsTransformer.java

```diff
@@ -67,17 +67,19 @@ private boolean requiresValidation(MethodInfo method) {
                 return true;
             }
         }
+
         // This method has no annotations of its own: look for inherited annotations
-        ClassInfo clazz = method.declaringClass();
         SimpleMethodSignatureKey signatureKey = new SimpleMethodSignatureKey(method);
```

Comment:
This could be moved further down because it is only needed when `validatedMethods` is not `null` or empty.